spec/lib/generators/module/module_generator_spec.rb

```diff
@@ -4,6 +4,14 @@
 require 'generators/module/module_generator'
 
 describe ModuleGenerator do
+  before(:all) d
```

Comment:
Typo here

Reply:
woops!  another reminder to run tests before commit/push, lol